### DISCUSSION TITLE
Annotate nodes?

### DISCUSSION THREAD

**Discussion by gefloh** [mysticaltech/terraform-hcloud-kube-hetzner#828] [Q&A] [answered]
Hi,

just a tiny (probably newbie) question: I see that I can label nodes during creation. Is it possible to also annotate them?

Thanks,
Gerhard

**Comment by mysticaltech** [comment, marked as answer]
@gefloh Not at this moment! PRs most welcome.

**Reply by gefloh** [reply]
Ok cool, will try to fiddle that out.

**Reply by gefloh** [reply]
Seems this feature is not wanted upstream: https://github.com/k3s-io/k3s/issues/2626. Exactly the use case why I wanted to use it too: For Longhorn.

**Reply by mysticaltech** [reply]
Ah yes, thanks for the info!
